src/app/components/administrator/dashboard/add-users/add-judges/add-judges.component.html

```diff
@@ -0,0 +1,56 @@
+<div class="flex-container">
+    <div class="top-options">
+        <h1>Agregar Juez</h1>
+        <h3>Crea una cuenta para un nuevo juez</h3>
+      </div>
+      <form class="add-form">
+        <mat-form-field class="flex-container" appearance="fill">
```

Comment:
The appearance of the inputs that we are using is outlined, should try to use those ones.